web/src/components/Button.tsx

```diff
@@ -113,3 +114,65 @@ export const button = cva(
 		},
 	},
 );
+
+export const button2 = cva(
```

Comment:
why do we need button2?
why not add another variant?
it does not make any sense

Reply:
Button had a relative attribute and I did not want to move relative to primary secondary destructive tetritary and repeat it four times so I created my button2. But I can change it

Reply:
no, please just add an override className, that's it. This is a perfect use case of using override class in cva.
Please delete this component, and use `button({intent: "primary", className: "absolute"})`